app/controllers/shows_controller.rb

```diff
@@ -58,4 +58,17 @@ def destroy
     Show.find(params[:id]).destroy
     redirect_to shows_path
   end
+
+  # migrating from protected attr to strong param
+  # standard found here: https://www.fastruby.io/blog/rails/upgrades/strong-parameters-migration-guide.html
+  private
+  def permit_new_show
+    params.permit :season
+    params.require :season
+  end
+
+  def permit_create_show
+    params.permit :show
+    params.require :show
```

Comment:
try taking this out too

Reply:
had to permit particular attributes of show otherwise a ForBiddenAttributes was correctly being thrown